src/SFA.DAS.FindEpao/InnerApi/Responses/GetStandardsListItem.cs

```diff
@@ -7,5 +10,7 @@ public class GetStandardsListItem
         public string Title { get; set; }
         public int Level { get; set; }
         public bool IntegratedApprenticeship { get; set; }
+        public string StandardVersions { get; set; }
```

Comment:
Think this needs to be an array to be in line with the spec.

Reply:
done